string capability
string preferred_provider
---
string bond_id